### ISSUE TITLE
May I ask which paper is this reproduced code?

### ISSUE THREAD

**Issue by Aurora-0f** [safwankdb/Automated-measurement-of-Fetal-head-circumference#3]
May I ask which paper is this reproduced code?

**Comment by bmiftah** [comment]
This seems to be implementation of pure U-net for fetal head segmentation and HC measurement, were you able to run it ?

**Comment by Aurora-0f** [comment]
> This seems to be implementation of pure U-net for fetal head segmentation and HC measurement, were you able to run it ?

Haven't tried it yet, have you tried it yet? Does it work?

**Comment by bmiftah** [comment]
I am trying it . It needs some cleaning and re-arrangement ... I try running it on colab but it  didn't finsh due to resource limit. I am now running on CPU and will see what happen.

**Comment by Aurora-0f** [comment]
Okay, can you please tell me how it goes when it's over?

**Comment by bmiftah** [comment]
Sure... If it works! I am just running the train.py part for now ... 

 Why don't you try running it ? Do you have GPU ?

**Comment by Aurora-0f** [comment]
> 
It's a bit bad, my computer's hard drive broke a few days ago, and it's being repaired to restore data.

**Comment by bmiftah** [comment]
oh ! Okay here is the weight displayed when I run it on GPU .. It didnt finish ..Epoch 4 running
Epoch finished ! Loss: -0.34960441468637204
Epoch finished ! Loss: -0.3810150877584385
End of epoch
Epoch 5 running
Epoch finished ! Loss: -0.3464887431905239
Epoch finished ! Loss: -0.3820047413246541
End of epoch
Epoch 6 running
Epoch finished ! Loss: -0.34739051468764676
Epoch finished ! Loss: -0.31917487360254115
End of epoch
Epoch 7 running
Epoch finished ! Loss: -0.3468326894542839
Epoch finished ! Loss: -0.31565151063701774
End of epoch
Epoch 8 running
Epoch finished ! Loss: -0.3478112958654573
Epoch finished ! Loss: -0.3164069368492199
End of epoch .... 
it goes like this for 17 epoch ... I don't know why the loss remain the same and also negative.

**Comment by Aurora-0f** [comment]
Oh, it doesn't look very good, I will try it with the GPU when my hard drive data is restored. thank you very much.

**Comment by bmiftah** [comment]
It is first run ... I may have mess some parts ...  it may work good.  hopefully the authors will share us some tips on how to use the code.

**Comment by Aurora-0f** [comment]
Yeah,I agree with you very much,！Hope the author can see your message.

**Comment by bmiftah** [comment]
If you wish to check another code , I can share you link.  But it is not based on any paper. It is sample implementation of similar problem using U-net .

**Comment by Aurora-0f** [comment]
> If you wish to check another code , I can share you link. But it is not based on any paper. It is sample implementation of similar problem using U-net .

Oh,thank you.I will try it later.

**Comment by sorryandbyebye** [comment]
> If you wish to check another code , I can share you link. But it is not based on any paper. It is sample implementation of similar problem using U-net .

hello，Do you ever overfit when using the unet?
